.fossa.yml

```diff
@@ -1,87 +1,12 @@
-# Generated by FOSSA CLI (https://github.com/fossas/fossa-cli)
-# Visit https://fossa.com to learn more
+# https://github.com/fossas/fossa-cli/blob/v3.6.11/docs/references/files/fossa-yml.md
 
-version: 2
-cli:
-  server: https://app.fossa.com
-  fetcher: custom
-  project: git@github.com:kubevirt/kubevirt.git
-analyze:
-  modules:
-  - name: virt-operator
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/virt-operator'
-    path: cmd/virt-operator
-    options:
-      modules-vendor: true
-      strategy: gomodules
-  - name: example-cloudinit-hook-sidecar
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/example-cloudinit-hook-sidecar'
-    path: cmd/example-cloudinit-hook-sidecar
-    options:
-      modules-vendor: true
-      strategy: gomodules
-  - name: fake-qemu-process
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/fake-qemu-process'
-    path: cmd/fake-qemu-process
-    options:
-      modules-vendor: true
-      strategy: gomodules
-  - name: dump
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/dump'
-    path: cmd/dump
-    options:
-      modules-vendor: true
-      strategy: gomodules
-  - name: virt-handler
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/virt-handler'
-    path: cmd/virt-handler
-    options:
-      modules-vendor: true
-      strategy: gomodules
-  - name: virt-launcher
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/virt-launcher'
-    path: cmd/virt-launcher
-    options:
-      modules-vendor: true
-      strategy: gomodules
-  - name: virtctl
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/virtctl'
-    path: cmd/virtctl
-    options:
-      modules-vendor: true
-      strategy: gomodules
-  - name: chroot
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/virt-chroot'
-    path: cmd/virt-chroot
-    options:
-      modules-vendor: true
-      strategy: gomodules
-  - name: virt-api
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/virt-api'
-    path: cmd/virt-api
-    options:
-      modules-vendor: true
-      strategy: gomodules
-  - name: example-hook-sidecar
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/example-hook-sidecar'
-    path: cmd/example-hook-sidecar
-    options:
-      modules-vendor: true
-      strategy: gomodules
-  - name: virt-controller
-    type: go
-    target: 'kubevirt.io/kubevirt/cmd/virt-controller'
-    path: cmd/virt-controller
-    options:
-      modules-vendor: true
-      strategy: gomodules
+version: 3
+
+project:
+  id: git@github.com:kubevirt/kubevirt.git
+  url: github.com/kubevirt/kubevirt
+
+target:
+  only:
+  - type: gomod
+    path: ./
```

Comment:
I see that a lot is being removed here in favor of a pretty compact new yaml.
Are these configs eventually the same?

Reply:
Since we are jumping two full major versions of FOSSA cli, you never know :)

But seriously, the old config explicitly configured the scanning by naming each module below the `analyze` section.

The new config now relies on auto discovery of the projects. I think this is an improvement, since you don't have to explicitly add new modules, which one can easily forget to do.

Reply:
Since we are jumping two full major versions of FOSSA cli, you never know :)

But seriously, the old config explicitly configured the scanning by naming each module below the `analyze` section.

The new config now relies on auto discovery of the projects. I think this is an improvement, since you don't have to explicitly add new modules, which one can easily forget to do.